invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/overridableRegistry/mapping.js

```diff
@@ -1,7 +0,0 @@
-// This file is part of InvenioRDM
```

Comment:
I think this was removed by mistake.

Same for the file below, it should be reverted, `invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/overridableRegistry/mapping.js`

Reply:
This is weird, the symlink got in? Thanks for spotting it! 👍🏼